config/examples/BigTreeTech/SKR Mini E3 1.2/Configuration_adv.h

```diff
@@ -1567,9 +1565,9 @@
 #define LIN_ADVANCE
 #if ENABLED(LIN_ADVANCE)
   //#define EXTRA_LIN_ADVANCE_K // Enable for second linear advance constants
-  #define LIN_ADVANCE_K 0.0    // Unit: mm compression per 1mm/s extruder speed
+  #define LIN_ADVANCE_K 0.22    // Unit: mm compression per 1mm/s extruder speed
```

Comment:
This should not be enabled by default. Users need to tune this setting on their own, which is why it's left at 0.0.

Reply:
The reason for reverting it to 0.22 was, that this is the default value:
https://github.com/MarlinFirmware/Configurations/blob/import-2.0.x/config/default/Configuration_adv.h#L1569

Should this value than also changed in the default file?